sql/hive/src/test/scala/org/apache/spark/sql/hive/client/HiveClientSuite.scala

```diff
@@ -53,7 +52,7 @@ class HiveClientSuite(version: String)
       for {
         ds <- 20170101 to 20170103
         h <- 0 to 23
-        chunk <- Seq("aa", "ab", "ba", "bb")
+        chunk <- Seq("11", "12", "21", "22")
```

Comment:
What is the reason we need to change so many lines in the test case? Could you a new test case, instead of changing the existing one?

Reply:
Thanks a lot for review :)
I did two things:
1. Pass resolved `Expression` to generate filter -- because in current change, I need data type of the attribute;
2. "aa" -> "11", "ab" -> "12", "ba" -> "21", "bb" -> "22" -- because I need to pass an integer to set up the filter(chunk=11) and do the cast then.

Reply:
The first point looks fine, for the second one, can we generate new data for your new test case?

Reply:
Thanks, I should refine the tests.